docs/Guide.TypeScriptIntegrationJest.md

```diff
@@ -0,0 +1,111 @@
+---
+id: Guide.TypeScriptJest
+title: Integrating TypeScript with Detox + Jest
+---
+
+## Usage
+
+### 0. Initialize Detox in your project
```

Comment:
0 is probably unnecessary. It's already mentioned in other guides

Reply:
While I agree, I think it needs to be in there since there are a few steps that need to be taken after following those guides. I'll trim that section, however.

Reply:
I've cut out some of the content here and added a sentence saying why it is necessary to have the tests in the e2e folder